src/app/people/export/get.controller.js

```diff
@@ -0,0 +1,35 @@
+const { getPeople } = require('../../../common/config/endpoints');
+const CookieModel = require('../../../common/models/Cookie.class');
+const personApi = require('../../../common/services/personApi');
+const logger = require('../../../common/utils/logger')(__filename);
+const createArrayCsvStringifier = require('csv-writer').createArrayCsvStringifier;
+
+const writePeopleAsCSVtoResponse = (res, people) => {
+    const csvStringifier = createArrayCsvStringifier({
+        header: ['Id', 'First Name', 'Last Name', 'Gender', 'DoB', 'Nationality', 'Doc Type', 'Doc Number', 'Doc Expiry', 'Doc Issuing State', 'Type']
+    });
+
+    const dateSlug = new Date().toISOString().slice(0, 10);
+    res.setHeader('Content-disposition', 'attachment; filename=people-' + dateSlug + '.csv');
+    res.setHeader('Content-Type', 'text/csv');
+    res.write(csvStringifier.getHeaderString());
+    res.write(csvStringifier.stringifyRecords(people));
+    res.end();
+}
+
+module.exports = (req, res) => {
+    logger.debug('In people export get controller');
+    const cookie = new CookieModel(req);
+
+    personApi.getPeople(cookie.getUserDbId(), 'individual')
+        .then((values) => {
```

Comment:
What happens here if there is an empty string?

Reply:
if there are no people the api returns a json empty array [] and the export is just the headers.